core/src/main/java/com/adobe/aem/commons/assetshare/content/renditions/AssetRenditions.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Asset Share Commons
+ *
+ * Copyright (C) 2019 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.adobe.aem.commons.assetshare.content.renditions;
+
+import com.adobe.aem.commons.assetshare.content.AssetModel;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.resource.ValueMap;
+import org.apache.sling.api.wrappers.ValueMapDecorator;
+import org.osgi.annotation.versioning.ProviderType;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ *
+ */
+@ProviderType
+public interface AssetRenditions {
+    String VAR_ASSET_PATH = "${asset.path}";
+    String VAR_ASSET_NAME = "${asset.name}";
+    String VAR_ASSET_EXTENSION = "${asset.extension}";
+    String VAR_RENDITION_NAME = "${rendition.name}";
+
+    /**
+     * @return a list of all registered AssetRenditionDispatchers in the system ordered in Descending order by service.ranking.
```

Comment:
This interface is marked as @ProviderType, and there is not an expectation that anyone else implements this. I think its better to leave this since the 99% would be using the provided impl, and it's worth knowing this detail.